src/external/modules/view-licences/controller.js

```diff
@@ -61,7 +61,8 @@ async function getLicenceDetail (request, reply) {
       licence_id: documentId,
       name: 'name' in request.view ? request.view.name : customName,
       licenceData: viewData,
-      back: `/licences/${documentId}`,
+      back: isInternal(request) ? `/admin/licences/${documentId}` : `/licences/${documentId}`,
```

Comment:
I think the isInternal() logic should be removed this sprint, the removed line in red is correct